.wp-env.json

```diff
@@ -1,4 +1,4 @@
 {
-	"core": "WordPress/WordPress",
-	"plugins": ["."]
+	"core": null,
```

Comment:
I think we can most likely remove this altogether, given it's the default. Not a blocker though.

Reply:
yes actually it doesn't make sense, if you want we can remove it in this PR since we are updating it, okay?

Reply:
```suggestion
```